content/blog/2023/08/30/2023-08-30-a-glimpse-of-the-future.adoc

```diff
@@ -0,0 +1,460 @@
+---
+layout: post
+title: "A glimpse of the future"
+tags:
+- jenkins
+- jdk21
+- risc-v
+authors:
+- gounthar
+opengraph:
+  image: /images/post-images/2023/08/30/2023-08-30-a-glimpse-of-the-future/a-glimpse-of-the-future.png
+discourse: true
+---
+
+image:/images/post-images/2023/08/30/2023-08-30-a-glimpse-of-the-future/image2.png[image,width=839]
+
+You've all seen it before: the bitter sting of botched predictions.
+Flying cars, nuclear ovens in the kitchen, killer robot dogs, and lunar living - all expected to be part of our daily lives by now.
+Well, some of these wild dreams did come to pass, but I'm not about to roll the dice and predict what our world will look like 50 years hence.
+Instead, let me paint a picture of Jenkins' future up until October 2024.
+
+Why am I being so specific about this end date, you ask?
+It's because I want to highlight two major projects that Jenkins might tackle before the leaves start to fall next year.
+
+One of these projects is the potential leap to JDK21, and we've got a proposed roadmap for that.
+Jenkins might make the switch to JDK21 in October 2024, as JDK11 could be left behind.
```

Comment:
I'd phrase it differently because the Java support plan is still  evolvling.
```suggestion
Jenkins will stop supporting JDK 11 in October 2024 when many Java providers (Eclipse Temurin, OpenJDK, ...) stop supporting Java 11.
```